client/transaction-pool/src/lib.rs

```diff
@@ -25,6 +25,8 @@ mod maintainer;
 pub mod error;
 #[cfg(test)]
 mod tests;
+#[cfg(feature = "test")]
+pub mod test_helpers;
```

Comment:
@tomusdrw needed to re-use some code from the transaction pool tests, not sure if this is the right approach,

Reply:
I'd call the module `mod testing`, but the feature `test-helpers`. I think this convention is more widespread in substrate. But looks good apart from that.